iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/port_introspection.hpp

```diff
@@ -62,12 +62,12 @@ class PortIntrospection
 
         struct PublisherInfo
         {
-            PublisherInfo() = default;
+            PublisherInfo() noexcept {};
```

Comment:
@elBoberido This will generate a warning in more strict builds:
https://rextester.com/live/WANPI61259
-Weverything
```
504430806/source.cpp:7:32: warning: extra ';' after member function definition [-Wextra-semi]
    PublisherInfo() noexcept {};
                               ^
```
the hint is at the comma

Reply:
@sculpordwarf thanks for the hint. I'll fix it with my next PR. I'm wondering why our CI didn't catch it